Resources/Prototypes/SS220/Entities/Clothing/OuterClothing/coats.yml

```
@@ -98,7 +98,17 @@
     sprite: SS220/Clothing/OuterClothing/Coats/captain_white.rsi
   - type: Clothing
     sprite: SS220/Clothing/OuterClothing/Coats/captain_white.rsi
-
+- type: entity
+  parent: ClothingUniformJumpskirtCapFormalDress
+  id: ClothingOuterBlackCoatCaptain
+  name: чёрное пальто капитана
+  description: Чёрное пальто? Такие выдаются лишь офицерам флота Nanotrasen.
+  suffix: За время в игре
+  components:
+  - type: Sprite
+    sprite: SS220/Clothing/OuterClothing/Coats/captain_blackcoat.rsi
+  - type: Clothing
+    sprite: SS220/Clothing/OuterClothing/Coats/captain_blackcoat.rsi
```

Прототипы лучше пустой строкой отделять чтобы каши не было
```suggestion

- type: entity
  parent: ClothingUniformJumpskirtCapFormalDress
  id: ClothingOuterBlackCoatCaptain
  name: чёрное пальто капитана
  description: Чёрное пальто? Такие выдаются лишь офицерам флота Nanotrasen.
  suffix: За время в игре
  components:
  - type: Sprite
    sprite: SS220/Clothing/OuterClothing/Coats/captain_blackcoat.rsi
  - type: Clothing
    sprite: SS220/Clothing/OuterClothing/Coats/captain_blackcoat.rsi

```

---

Также имя, описание и суффикс лучше сделать через локализацию (знаю что у некоторых прототипов тут на русском прописано, не уследили значит)